tests/columns/test_check.py

```diff
@@ -8,12 +8,54 @@
 
 
 class CheckSchema(dy.Schema):
-    a = dy.Int64(check=lambda col: (col < 5) | (col > 10))
-    b = dy.String(min_length=3, check=lambda col: col.str.contains("x"))
+    a = dy.Int64(check={"check": lambda col: (col < 5) | (col > 10)})
+    b = dy.String(min_length=3, check={"check": lambda col: col.str.contains("x")})
 
 
 def test_check() -> None:
     df = pl.DataFrame({"a": [7, 3, 15], "b": ["abc", "xyz", "x"]})
     _, failures = CheckSchema.filter(df)
     assert validation_mask(df, failures).to_list() == [False, True, False]
     assert failures.counts() == {"a|check": 1, "b|min_length": 1, "b|check": 1}
+
+
+def test_check_names() -> None:
+    def str_starts_with_a(col: pl.Expr) -> pl.Expr:
+        return col.str.starts_with("a")
+
+    def str_end_with_z(col: pl.Expr) -> pl.Expr:
+        return col.str.ends_with("z")
+
+    class MultiCheckSchema(dy.Schema):
```

Comment:
Can you add a column where we pass checks as a list of lambdas? That'll maintain 100% code coverage (see [here](https://app.codecov.io/gh/Quantco/dataframely/pull/19?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Quantco#c1e424ec368c65c706a8a51dcc3fa5b3-R111))